code/_helpers/unsorted.dm

```diff
@@ -1242,6 +1242,13 @@ var/list/FLOORITEMS = list(
 /proc/crash_with(msg)
 	CRASH(msg)
 
+//gives us the stack trace from CRASH() without ending the current proc.
+/proc/stack_trace(msg)
```

Comment:
Why is this needed both as a global proc and as a datum's?

Reply:
Keep that from original source.
Actually agree with you. We need only `CRASH(msg)` or `crash_with(msg)` for real.